message MSG_CLIENT_DM_USER_CREATE
{
    optional string token = 1;
    optional string domainName = 2;
    optional string groupName = 3;
    optional string userName = 4;
    optional string password = 5;
    optional uint32 authority = 6;
}

message MSG_DM_CLIENT_USER_CREATE_ACK
{
    optional int32 statusCode = 1;
}